helix-core/src/history.rs

```diff
@@ -133,6 +133,18 @@ impl History {
         Some(&self.revisions[last_child.get()].transaction)
     }
 
+    pub fn last_edit_pos(&mut self) -> Option<usize> {
+        let current_revision = &self.revisions[self.current];
+        if let Some(op) = &current_revision.transaction.changes().changes.first() {
+            if let crate::Operation::Retain(pos) = op {
```

Comment:
Instead of looking at operations, use  `changes_iter` https://github.com/helix-editor/helix/blob/9e8f27786dce957869bd157555e2ab5cfee3b019/helix-core/src/transaction.rs#L533-L535

 to get back the `(from, to, change)` tuple, and use `from` as the pos